build.xml

```diff
@@ -30,6 +30,9 @@
 
     <property name="webapp.dir" value="${main.dir}/webapp" />
     <property name="webapp.src.dir" value="${main.dir}/src" />
+    <property name="table-schema.src.dir" value="${basedir}/tableschema-java/src/main/java/" />
```

Comment:
For some reason when I try to build on Windows it cannot find this path and complains.

Reply:
Did you do git clone --recursive https://github.com/OpenRefine/OpenRefine

Reply:
Again we should really use jars to avoid that!

Reply:
@jackyq2015 change that... I don't like this setup... jars preferred.  So what if its a bit of maintenance.  This is more pain than its worth.